tests/core/pyspec/eth2spec/test/helpers/attestations.py

```diff
@@ -373,6 +374,26 @@ def temp_participants_filter(comm):
     return attestations
 
 
+_prep_state_cache_dict = LRU(size=10)
+
+
+def cached_prepare_state_with_attestations(spec, state):
+    """
+    Cached version of prepare_state_with_attestations,
+    but does not return anything, and does not support a participation fn argument
+    """
+    # If the pre-state is not already known in the LRU, then take it, make it leaking, and put it in the LRU.
```

Comment:
what does "make it leaking" mean?

Reply:
to run those state transitions to get validators leaking balance, like the tests are for. I might have messed up here and copied the comment from the other cache. Whoops.